scripts/.markdownlint.yaml

```
@@ -0,0 +1,12 @@
+default: true
+
+ul-style:
+  style: asterisk
+
+no-multiple-blanks:
+  maximum: 1
+
+line-length: false
+no-inline-html: false
+first-line-h1: false
+emphasis-style: false

```

```suggestion
emphasis-style: false
heading-style: atx
no-reversed-links: true
no-emphasis-as-heading: true
no-space-in-emphasis: true
no-alt-text: true
```

Deze lijken mij ook handig om toe te voegen; Vooral de laatste is een een voorwaarde voor toegankelijkheid.

---

Zoals offline besproken staan deze regels al aan met `default: true`. De atx style heb ik wel overgenomen.